open-craft/demo-mfe
# [STAR-2442] Upgrade dependencies to match upstream

This PR upgrades the dependencies in this repository to match upstream, with one exception-- upgrading paragon to the latest fix release. The upstream version breaks.

**JIRA tickets**: https://tasks.opencraft.com/browse/STAR-2442

**Testing instructions**:

1. Have a working and running devstack
2. Install [NVM](https://github.com/nvm-sh/nvm)
3. Create a new terminal session to get the new env variables from your profile
4. cd into the repo
5. `nvm install`
6. `nvm use`
7. `npm install -g npm@8.6.0`
8. `npm install`
9. `npm run start`
10. Visit http://localhost:8080/catalog/ and verify things work.
